CONTRIBUTING.md

```diff
@@ -51,3 +51,35 @@ JSON is formatted using Prettier.
 
 **Avoid code formatting in generated JSON files**: files generated by tools, such as `package.json` that regularly updated using `npm install`, should be excempt from automatic formatting. Configure these files in `.prettierignore`.
+
+## Contributing as a designer or content specialist
+
+We encourage contributions from designers, copywriters and other stakeholders. There are many different ways to make a useful contribution, from the addition of (UX) documentation, new design components to fixing typing errors.
+
+In order to contribute you must have a [GitHub account](https://github.com/join).
+
+### Using GitHub
+
+GitHub allows the NL Design System to be maintained in the open. Hosting the content on GitHub has several benefits; all the content is freely available to everything in a central place and can accept contributions from everyone. Version control means that a full history is kept of the progress of and changes to the design system. Changes can be reverted when needed.
+
+The easiest way to work with GitHub is through the [GitHub Desktop application](https://desktop.github.com/), available for both Mac OS-X and Windows.
+
+For both new additions and changes to existing components or documentation you first need to create a ‘branch’. A branch duplicates the ‘repository’ (all content of NLDS), and allows you to make changes or additions safely in isolation without impacting the rest of the project.
+
+Once your work is finished you are able to ‘commit’ your changes first to your branch. It is important to describe as clear and concise as possible what you have changed or added in the commit message. This message will be displayed to reviewers and all other contributors of the design system.
+
+NLDS uses [Conventional Commits](https://www.conventionalcommits.org/) in order for commit messages to follow a consistent pattern.
+
+After committing your changes to your branch you need to merge them to the main repository by clicking on the ‘Publish branch’ button. You have to make an additional ‘pull request’ to let reviewers know about your changes.
+
+Before an addition or change is approved and added to the design system it will first go through a review phase. If changes are needed, you will automatically receive an email on the address associated with your GitHub account.
+
+### Writing documentation
+
+Our documentation is written in [Markdown](https://daringfireball.net/projects/markdown/), an easy to write and read text format that can easily be converted to other formats. Markdown allows documents to be consistently formatted and structured.
+
+GitHub uses its own ‘flavor’ of Markdown. [See this list for an overview of all the formatting options](https://guides.github.com/features/mastering-markdown/) allowed in our documentation.
+
+### Design components
+
+We currently support Figma-based components to be included in out documentation.
```

Comment:
out → our

Also: recently I noticed I am trying to avoid "we" and "our" a bit, since I felt it not as inclusive to community members who a haven't contributed commits yet but intend to do so. And also not to imply the existence of a core development/support team. I'm not entirely sure about this approach yet, butI find that "our" can frequently be left out or you can just refer to "NL Design System" or "the project".

So it is not my recommendation per se, I am just sharing an observation and gut feeling.

Reply:
Good point, will rephrase!